common/cauthdsl/cauthdsl.go

```diff
@@ -76,7 +76,7 @@ func compile(policy *cb.SignaturePolicy, identities []*mb.MSPPrincipal) (func([]
 				}
 				err := sd.SatisfiesPrincipal(signedByID)
```

Comment:
I'm not sure we should do this. The way policy is evaluated is by exhaustive search. This means that if you have an endorsement policy of AND(Org1, Org2) then if you have signatures from 2 and then from 1, you will see a warning despite the EP being satisfied.